config/seata-config-core/src/main/java/io/seata/config/ConfigType.java

```diff
@@ -66,6 +66,6 @@ public static ConfigType getType(String name) {
                 return configType;
             }
         }
-        throw new IllegalArgumentException("illegal type:" + name);
+        throw new IllegalArgumentException("not support config type: " + name);
```

Comment:
How about do not change here?

Reply:
Because this is the getType method of the Enum class, the reason for the exception here is that the input parameter is illegal.